src/platforms/android/androidcontroller.cpp

```diff
@@ -215,11 +233,17 @@ bool AndroidController::VPNBinder::onTransact(int code,
     Q_UNUSED(flags);
 
     QJsonDocument doc;
-    QString logs;
+    QString buffer;
     switch (code) {
     case EVENT_INIT:
         logger.log() << "Transact: init";
-        emit m_controller->initialized(true, false, QDateTime());
+         buffer = QTextCodec::codecForMib(106)->toUnicode(data.readData());
```

Comment:
Can you make a separate function to do this step. Otherwise I'll ask you what 106 is each time.